### DISCUSSION TITLE
Containers status unhealthy - after reboot

### DISCUSSION THREAD

**Discussion by wisdomlight** [nextcloud/all-in-one#4251] [Questions] [answered]
I moved the machine to a new location.
Fired it up - there is no internet connection at the moment.
All containers have unhealthy status.
How do I find out what's the cause?
Thank you

**Comment by szaimen** [comment, marked as answer]
> there is no internet connection at the moment.

This is your issue

**Reply by wisdomlight** [reply]
Thank you @szaimen
